src/test/utils/index/nc_index.js

```diff
@@ -0,0 +1,35 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
```

Comment:
why this is a new file?

Reply:
notice that the old nc_index file appear to be deleted. I guess that because there were a lot of changes in the file. all of the requires were changed. github detected this to be a new file instead of a rename